inventory/host_vars/staging.openfoodnetwork.org.au/config.yml

```diff
@@ -18,3 +18,6 @@ unicorn_timeout: 120
 # Size in bytes. You can also use units like 1G, 512MiB or 1000KB. See: `man fallocate`
 # The default is `false`, not installing a swapfile.
 swapfile_size: 1G
+
+# Images settings
+attachment_path: "public/images/spree/products/:id/:style/:basename.:extension"
```

Comment:
same here, I'd just use the default (public/spree/products/:id/:style/:basename.:extension) and configure S3 accordingly.

Reply:
I like the idea of standardising but since we have non-standard production servers it would be good to also test with non-standard staging servers. When I tested your pull request removing the image settings, au-staging had broken images. That's exactly the code under test. I think that adding this config here is the easiest way to test that it works.

I would still be happy to standardise this later on. I think it will be easier to change this through the config without caching once we have the new code deployed. But if you feel very strongly about it then we can standardise everything now and find a different way to test this on staging.

Reply:
Hello Maikel, it's not a strong opinion, just an opinion :-)
We can change these settings on staging servers to make the test yes, but I'd keep the configs in staging without all this messy stuff from production. My vote goes to change these things for the test as needed but afterwards keep the staging servers and GH as is without this PR.
Basically, I'd keep it simple in staging and work to standardize the settings in production instead of copying the mess from production.

Maybe another dev can decide @Matt-Yorkley or @sauloperez

Reply:
Okay, you convinced me that we should standardise most of it. I'm still wondering if it could be good to keep one staging server different, au-staging for example. But as you said, we can change the config during dev tests. It's just a bit more complicated moving images in the bucket as well to see if the test works. My current thought is:

* Sort out all the non-standard configs except au-staging.
* Test openfoodfoundation/openfoodnetwork#5729 with a modified config.
* Then standardise au-staging as well.

In that case we won't need this pull request. I would still leave it open as long as we haven't sorted all the configs. It can be our todo list.

Reply:
ok, let's do it :+1:

Reply:
I adjusted all server configs except au-staging.

Reply:
:+1: